beeline/src/java/org/apache/hive/beeline/Commands.java

```diff
@@ -1063,13 +1057,17 @@ private boolean executeInternal(String sql, boolean call) {
             showRemainingLogsIfAny(stmnt);
           }
         }
+        if (rs != null) {
+          rs.close();
```

Comment:
Closing the result set can throw a `SQLException` in which case it will skip over the `Statement#close()` method.